schema.json

```diff
@@ -0,0 +1,514 @@
+{
+  "$schema": "http://json-schema.org/LivingDocumentation",
+  "title": "Intermediary JSON for LivingDocumetation",
+  "type": "array",
+  "items": {
+    "type": "object",
+    "properties": {
+      "FullName": {
+        "type": "string"
+      },
+      "Type": {
+        "type": "number"
+      },
+      "Modifiers": {
+        "type": "number"
+      },
+      "BaseTypes": {
+        "type": "array",
+        "items": {
+          "type": "string"
+        }
+      },
+      "Fields": {
+        "$ref": "#/$defs/fields"
+      },
+      "Constructors": {
+        "$ref": "#/$defs/constructors"
+      },
+      "Methods": {
+        "$ref": "#/$defs/methods"
+      },
+      "Properties": {
+        "$ref": "#/$defs/$properties"
+      },
+      "Attributes": {
+        "$ref": "#/$defs/attributes"
+      },
+      "EnumMembers": {
+        "$ref": "#/$defs/enumMembers"
+      },
+      "Events": {
+        "$ref": "#/$defs/events"
+      },
+      "DocumentationComments": {
+        "$ref": "#/$defs/documentationComments"
+      }
+    },
+    "additionalProperties": false,
+    "required": [
+      "FullName"
+    ]
+  },
+  "$defs": {
+    "statements": {
+      "type": "array",
+      "items": {
+        "anyOf": [
+          {
+            "$ref": "#/$defs/forEach"
+          },
+          {
+            "$ref": "#/$defs/if"
+          },
+          {
+            "$ref": "#/$defs/switch"
+          },
+          {
+            "$ref": "#/$defs/invocation"
+          },
+          {
+            "$ref": "#/$defs/assignment"
+          },
+          {
+            "$ref": "#/$defs/return"
+          }
+        ]
+      }
+    },
+    "forEach": {
+      "type": "object",
+      "properties": {
+        "$type": {
+          "const": "LivingDocumentation.ForEach, LivingDocumentation.Statements"
+        },
+        "Expression": {
+          "type": "string"
+        },
+        "Statements": {
+          "$ref": "#/$defs/statements"
+        }
+      },
+      "additionalProperties": false,
+      "required": [
+        "Expression"
+      ]
+    },
+    "if": {
+      "type": "object",
+      "properties": {
+        "$type": {
+          "const": "LivingDocumentation.If, LivingDocumentation.Statements"
+        },
+        "Sections": {
+          "type": "array",
+          "items": {
+            "Condition": {
+              "type": "string"
+            },
+            "Statements": {
+              "$ref": "#/$defs/statements"
+            }
```

Comment:
This probably needs to get a `"type": "object"` and then these need to go in `"properties"`.